src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/Tool/MsAADTool.cs

```diff
@@ -108,8 +108,12 @@ internal async Task<IList<Application>> GetApplicationsAsync()
 
             if (applicationList.Any())
             {
-                Organization? tenant = await GraphObjectRetriever.GetTenant();
-                if (tenant != null && tenant.TenantType.Equals("AAD B2C", StringComparison.OrdinalIgnoreCase))
+                var tenant = await GraphObjectRetriever.GetTenant();
+                if (tenant is null)
+                {
```

Comment:
do we want no error message for VS or commandline before we exit?

Reply:
Moved the failure handling to inside GetTenant(), so it will exit and log if it fails there

Reply:
sgsg